models/featureFlags.js

```diff
@@ -0,0 +1,137 @@
+const firestore = require('../utils/firestore')
+const featureFlagModel = firestore.collection('featureFlags')
+const userModel = require('./users')
+
+/**
+ * Fetch all tasks
+ *
+ * @return {Promise<featureFlags|Array>}
+ */
+const fetchFeatureFlag = async () => {
+  try {
+    const snapshot = await featureFlagModel.get()
+    const featureFlags = []
+    snapshot.forEach((doc) => {
+      featureFlags.push({
+        id: doc.id,
+        ...doc.data()
+      })
+    })
+    const users = []
+    const result = {}
+
+    featureFlags.forEach((item) => {
+      if (!users.includes(item.owner)) { users.push(item.owner) }
+    })
+
+    const getImage = async (usersData) => {
```

Comment:
this function is not needed. You can directly call `fetchUserImage` in the loop below.